open3d_slam/open3d_slam/src/helpers.cpp

```diff
@@ -79,9 +77,9 @@ class point_cubic_id {
 };
 
 }  // namespace
-
+// Treat black as no color
 bool isValidColor(const Eigen::Vector3d& c) {
```

Comment:
we have a duplication of implementation. Implement one method in terms of the other.